src/caste_chyby/caste_chyby.md

```diff
@@ -47,6 +47,34 @@ a && 4; // `true`
 if (a > 5 & a < 6) {}
 ```
 
+### Středník za `for`, `while` nebo `if`
+Středník za `for`, `while` nebo `if` by ve většině případů neměl být, protože se na jeho místě vytvoří prázdný blok kódu.
```

Comment:
Navrhuji to přeformulovat následovně:
~~~
Příkazy `for`, `while` nebo `if` za svou uzavírací závorkou `)` očekávají jeden příkaz:
```c
if (a > b) printf("%d", a);
```
nebo blok s příkazy:
```c
if (a > b) {
    printf("%d", a);
    ...
}
```

Pokud však za závorku dáte za závorku středník (`;`), tak překladač to pochopí jako prázdný příkaz, který nic nedělá.
~~~